datasets/hate_offensive/README.md

```diff
@@ -0,0 +1,149 @@
+---
+
+YAML tags:
+annotations_creators:
+- crowdsourced
+language_creators:
+- machine-generated
+languages:
+- en
+licenses:
+- mit
+multilinguality:
+- monolingual
+size_categories:
+- n<1K
```

Comment:
Hm this does not look right (train split has 24K+ samples according to `dataset_infos.json`)
Should probably be `10k<n<100k`